variable/technology/power-plant.yaml

```diff
@@ -0,0 +1,756 @@
+# List of variables related to power plants
+# Variables related to the production cost of a power plant
+# Note that:
+# linear   term cost   [€/GWh] = Linear Var Cost [Mcal/GWh] * Price|Primary Energy [€/Mcal] + Operation and Maintenance Variable Cost [€/MWh] * 1e-3
+# constant term cost   [€/h]   = Constant Var Cost [Mcal/h] * Price|Primary Energy [€/Mcal]
+Constant Var Cost|Electricity|Coal:
```

Comment:
This variable in plan4res in called Fixed Cost but I havent included it here as in my dataset I do not have values for the fixed cost. We should only have one variable for those both terms. Both are fine for me